spk/dnscrypt-proxy/src/service-setup.sh

```diff
@@ -117,25 +117,25 @@ service_prestart () {
 }
 
 service_poststop () {
-    echo "After stop (service_poststop)" >> "${INST_LOG}"
+    echo "After stop (service_poststop)"
     blocklist_cron_uninstall
     forward_dns_dhcpd "no"
 }
 
 service_postinst () {
-    echo "Running service_postinst script" >> "${INST_LOG}"
-    mkdir -p "${SYNOPKG_PKGDEST}"/var >> "${INST_LOG}" 2>&1
+    echo "Running service_postinst script"
+    mkdir -p "${SYNOPKG_PKGDEST}"/var
     if [ ! -e "${CFG_FILE}" ]; then
         # shellcheck disable=SC2086
-        cp -f ${EXAMPLE_FILES} "${SYNOPKG_PKGDEST}/var/" >> "${INST_LOG}" 2>&1
-        cp -f "${SYNOPKG_PKGDEST}"/offline-cache/* "${SYNOPKG_PKGDEST}/var/" >> "${INST_LOG}" 2>&1
-        cp -f "${SYNOPKG_PKGDEST}"/blocklist/* "${SYNOPKG_PKGDEST}/var/" >> "${INST_LOG}" 2>&1
+        cp -f "${EXAMPLE_FILES}" "${SYNOPKG_PKGDEST}/var/"
```

Comment:
I'm not sure this works. `${EXAMPLE_FILES}` includes a glob: `example-*`

Reply:
```suggestion
        cp -f ${EXAMPLE_FILES} "${SYNOPKG_PKGDEST}/var/"
```

This is a regression, otherwise I'm happy to merge as is